init-letsencrypt.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+read -p "Enter your domain name (example.com) : " DOMAIN_NAME
```

Comment:
@yashdusing this is not clear. Should we enter space separated domain names? Please add appropriate comment

Reply:
no. So in our case it would be `cloudcv.org`

Reply:
this is not correct. We already host our production evalai at `evalai.cloudcv.org` we should not accept suffix. We should accept complete domain names

Reply:
Changed now. Accepting backend & frontend separately